example.rego

```diff
@@ -74,3 +81,13 @@ users := {
 	"bob": {"readOnly": true},
 	"alice": {"readOnly": false},
 }
+
+allowed_registries := [
+	"public.ecr.aws/",
+	"busybox",
```

Comment:
This too should have a trailing /, or else there's always the risk that someone registers `busyboxevilhacker` and we'll consider it a an allowed registry.

Reply:
Since docker allows "busybox" as image I tested some more and changed the policy to allow if the image matches explicitly or has registry prefix.
@anderseknert do you think this is a ok solution, or do you have any tips on a better one? :) Thanks